changelog.txt

```diff
@@ -1,3 +1,7 @@
+Version 1.0.15
+------------
+- TODO
```

Comment:
@rpdome So what are we ultimately shipping?

- Fix for [msal#915](https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/915)
    * Incorrect id_token returned for B2C app with multiple policies
- Fix for [msal#916](https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/916)
    * WebView calls `loadUrl` multiple times over lifecycle
- Fix for [msal#921](https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/921)
    * WebView displays error when connectivity lost
- Fix for [msal#924](https://github.com/AzureAD/microsoft-authentication-library-for-android/issues/904)
    * AT caching logic change for scope intersection
- TBD: Enhancement for WebView zoom control toggle**

** Should this force us to go `1.3.0`?

Reply:
for ** I think we don't have to, given that this is an optional field in the config object.

Reply:
@rpdome From the below criteria (taken from [semver](https://semver.org/)), I'd still interpret `**` as an instance of `#2`

```
Given a version number MAJOR.MINOR.PATCH, increment the:
1. MAJOR version when you make incompatible API changes,
2. MINOR version when you add functionality in a backwards compatible manner, and
3. PATCH version when you make backwards compatible bug fixes.
```

Reply:
I'm fine with that :D